packages/transformers/graphql/src/GraphQLTransformer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import {Transformer} from '@parcel/plugin';
+
+const IMPORT_RE = /^# *import +['"](.*)['"] *;? *$/;
+const NEWLINE = /\r\n|[\n\r\u2028\u2029]/;
+
+export default new Transformer({
+  async transform({asset, options, resolve}) {
+    // Peer dependency of graphql-tag
+    await options.packageManager.require('graphql', asset.filePath);
```

Comment:
This should just ensure graphql is installed and auto-install if needed. Lemme know if there's an other function that does this without actually doing a require as that would probably be slightly less overhead

Reply:
Didn't know that did an install. But just checked and it did, thanks for the tip